sapi/litespeed/lsapi_main.c

```diff
@@ -865,7 +865,7 @@ static int lsapi_activate_user_ini_finally(_lsapi_activate_user_ini_ctx *ctx,
     return rc;
 }
 
-static int lsapi_activate_user_ini(TSRMLS_D)
+static int lsapi_activate_user_ini( void )
```

Comment:
It looks like this code is compatible with multiple versions, so it should probably not be changed.